django/forms/forms.py

```diff
@@ -200,11 +200,11 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
             html_class_attr = ''
             bf = self[name]
             # Escape and cache in local variable.
-            bf_errors = self.error_class([conditional_escape(error) for error in bf.errors])
+            bf_errors = self.error_class(bf.errors)
             if bf.is_hidden:
                 if bf_errors:
                     top_errors.extend(
-                        [_('(Hidden field %(name)s) %(error)s') % {'name': name, 'error': str(e)}
+                        [conditional_escape(_('(Hidden field %(name)s) %(error)s') % {'name': name, 'error': str(e)})
```

Comment:
Did you see any test failures without the `conditional_escape` here? It might be needed and some test coverage is missing...
Can you investigate if something has changed since the `conditional_escape()` introduction in bcd63cbfb0590a2e2bed3e4beab3f467279ad3db or if it was never needed?

Reply:
@timgraham tests are passing even without `conditional_escape()`. That happens because `ErrorList` (`top_errors`) is automatically escaped when casted to string. I've checked both master and bcd63cb implementation and `__str__`/`__unicode__` are escaping error list on both.  This behaivour is covered with tests ([`test_utils.py`](https://github.com/django/django/blob/c651331b34b7c3841c126959e6e52879bc6f0834/tests/forms_tests/tests/test_utils.py#L112)).
I think it's safe to remove `conditional_escape` from this part completely.